hello/say_hello.py

```diff
@@ -8,3 +8,11 @@ def greet(name):
 
```

Comment:
Please revert say_hello.py to how it was before, and remove contents that you added.